golang/vaas/v2/examples/file-verdict-request/main.go

```diff
@@ -66,7 +66,7 @@ func main() {
 	defer analysisCancel()
 
 	// Request a verdict for a specific file (replace "path-to-your-file" with the actual file path)
-	result, err := vaasClient.ForFile(analysisCtx, "1GB.testfile")
+	result, err := vaasClient.ForFile(analysisCtx, "path-to-your-file")
```

Comment:
???

Reply:
Someone tested with 1GB.testfile and commited the file name. Comment says "path-to-your-file".